sparks_pif_converters/SEM/sem_to_pif.py

```diff
@@ -41,16 +41,26 @@ def s3000_metadata_to_pif(closed_txt):
 def convert_tif_to_jpeg(tiff_image):
 
     im = Image.open(tiff_image)
-    im.save(tiff_image.replace(".tif", ".jpeg"), "JPEG")
+    jpeg_path = tiff_image.replace(".tif", ".jpeg")
+    im.save(jpeg_path, "JPEG")
     print("CONVERTED: %s to %s" % (tiff_image, tiff_image.replace(".tif", ".jpeg")))
 
+    return jpeg_path
 
-def image_to_pif(image, file_type):
-    print("PARSING: "+image)
+def image_to_pif(image_path):
+
+    print("PARSING: {}").format(image_path)
+
+    file_type = image_path.split(".")[-1]
+
+    if file_type == "tif":
+        jpeg_path = convert_tif_to_jpeg(image_path)
+        image_to_pif(jpeg_path)
```

Comment:
I think you want a `return` after this line.  Otherwise, it will continue with the next block